datasafe-rest-impl/pom.xml

```diff
@@ -48,6 +53,16 @@
             <groupId>de.adorsys</groupId>
             <artifactId>datasafe-storage-impl-s3</artifactId>
             <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.amazonaws</groupId>
+                    <artifactId>aws-java-sdk-s3</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.amazonaws</groupId>
+                    <artifactId>aws-java-sdk-core</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
What is the reason to exclude com.amazonaws dependencies and then add them again?
If there are maven-dependency-plugin warnings, please add ignoredUsedUndeclaredDependencies in plugin configuration.